### ISSUE TITLE
Can't use the `verify()` test function on my main :app module when I need to ignore some types that are part of my sub-modules and have internal modifier

### ISSUE THREAD

**Issue by StylianosGakis** [InsertKoinIO/koin#1573]
**Is your feature request related to a problem? Please describe.**
I am trying to incorporate the `verify()` test task in my unit tests to ensure that my graph is correct.
When doing so, I've figured out that the types that are not part of the graph but I rely on the call-site to provide them using `parametersOf()` need to be ignored by this task but passing them as a list to the verify function. However, this is impossible for types that are only present in modules and are internal to those modules by default.

**Describe the solution you'd like**
I am not sure what a solution would look like. but there's definitely other folks who have a similar setup and I am curious what steps they've taken to get this work for me.

**Describe alternatives you've considered**
I could potentially create a list of Class<*> inside each module which has this problem, and expose that publicly so that this test would be able to get it per-module and add it to this list of ignored classes, but this certainly feels like quite a lot of ceremony. Is this what I should be doing in this case?
And I worry that this would be misused by other callers of that module. Maybe this list of classes can exist only in the test directory.

**Target Koin project**
The testing artifact

**Comment by arnaudgiuliani** [comment]
Yep, still a limitation of verify() API. Work in progress on it, to help make it better

**Comment by StylianosGakis** [comment]
Hey Arnaud, I am not sure I follow, is there some workaround in the latest koin versions on how to achieve this now? Or do you suggest that with 1.3.0 and by using annotation based setup you'll be able to do this in a simple way?

**Comment by JajaComp** [comment]
You can check this pull reuest - https://github.com/InsertKoinIO/koin/pull/1854

**Comment by StylianosGakis** [comment]
While that works, it's so much manual setup that I am not even sure it's worth the effort of maintaining it forever, while also making sure to remove the names of the ones that may no longer be relevant

**Comment by arnaudgiuliani** [comment]
Let's make this outdated as the coming Koin Compiler Plugin will ensure you safety with the DSL too 👍
